examples/testapp/src/components/RpcMethods/RpcMethodCard.tsx

```diff
@@ -174,9 +174,20 @@ export function RpcMethodCard({ format, method, params, shortcuts }) {
                 <AccordionPanel pb={4}>
                   <HStack spacing={2}>
                     {shortcuts.map((shortcut) => (
-                      <Button key={shortcut.key} onClick={() => submit(shortcut.data)}>
-                        {shortcut.key}
-                      </Button>
+                      <HStack key={shortcut.key} spacing={1}>
+                        <Button onClick={() => submit(shortcut.data)}>
+                          {shortcut.key}
+                        </Button>
+                        {shortcut.data.message && (
+                          <Button
+                            onClick={() => navigator.clipboard.writeText(JSON.stringify(shortcut.data.message, null, 2))}
```

Comment:
IIRC I think `writeText` can throw. Should we create a small util to capture the exception and maybe toast when its successful?

Reply:
I'd rather leave as is, I manually checked each copy button - each copy button is only generated from hardcoded examples we build.